### ISSUE TITLE
Status Update

### ISSUE THREAD

**Issue by tpnsolutions** [jaredmauch/mailman2-python3#6]
Has the code been converted to python 3 as the project indicates? What works, what doesn't? Which version of Mailman 2 is this based on?

**Comment by nicomen** [comment]
Seems to be based on latest stable version 2.1.39 https://github.com/jaredmauch/mailman2-python3/commit/40eccb3a4a11bbbe302c38f6026751aae97e827d

**Comment by jaredmauch** [comment]
yes, it's based on the latest stable and I'm happy to add other maintainers

**Comment by tpnsolutions** [comment]
thanks!

**Comment by thegushi** [comment]
My current experience is: it builds and I'm likely to attempt to put some time into it and attempt to get a "real" mailing list going on my own machines, and put it through its paces, before testing at the dayjob which is also a mailman2 shop with 20 years of history.

Testing is happening, if you want to try it out.

CPanel have their own builds that they are using, and I've proposed to try to work together more closely to tease apart what's general-fixes and what's cpanel-stack-specific.

Hope that helps.
